composer.json

```diff
@@ -37,8 +37,9 @@
         "phpstan/phpstan-symfony": "^1.3",
         "doctrine/orm": "^2.6 || ^3",
         "doctrine/persistence": "^2.0 || ^3 ",
-        "symfony/phpunit-bridge": "^6.3 || ^7",
-        "symfony/var-exporter": "^5.4 || ^6 || ^7"
+        "symfony/console": "^6.4.17",
```

Comment:
Had to add this constraint because of an indirect deprecation that caused the prefer-lowest job to fail.

Reply:
So we're not testing with Console 7? That would be really bad.

Reply:
Oh yeah sorry, I will add it.